safe_transaction_service/history/views.py

```diff
@@ -175,6 +176,75 @@ def get_queryset(self):
         return SafeMasterCopy.objects.relevant()
 
 
+class SafeDeploymentsView(ListAPIView):
+    """
+    Returns a list of safe deployments by version.
+    """
+
+    serializer_class = serializers.SafeDeploymentSerializer
+    pagination_class = None  # Don't show limit/offset in swagger
+
+    _schema_version_param = openapi.Parameter(
+        "version",
+        openapi.IN_QUERY,
+        type=openapi.TYPE_STRING,
+        default=None,
+        description="Filter by Safe version",
+    )
+    _schema_contract_param = openapi.Parameter(
+        "contract",
+        openapi.IN_QUERY,
+        type=openapi.TYPE_STRING,
+        default=None,
+        description="Filter by Safe contract name",
+    )
+
+    @swagger_auto_schema(
+        responses={404: "Provided version does not exist"},
+        manual_parameters=[
+            _schema_version_param,
+            _schema_contract_param,
+        ],
+    )
+    @method_decorator(cache_page(60))  # 60 seconds
+    def get(self, request):
+        """
+        Get current indexing status for ERC20/721 events
+        """
+        filter_version = self.request.query_params.get("version")
+        filter_contract = self.request.query_params.get("contract")
+
+        if filter_version and filter_version not in safe_deployments.keys():
+            return Response(status=status.HTTP_404_NOT_FOUND)
+
+        versions = [filter_version] if filter_version else list(safe_deployments.keys())
+        chain_id = get_chain_id()
+        data_response = []
+        for version in versions:
+            contracts = []
+            if not filter_contract:
```

Comment:
```suggestion
            if filter_contract:
```

Just a personal preference. I think if you are checking the attribute then you should do something about it now and not on the `else` branch. This could prevent issues if we are adding an `elif` in the future